cpp/log/cert.cc

```diff
@@ -274,8 +274,16 @@ Cert::Status Cert::HasExtension(int extension_nid) const {
     return ERROR;
   }
 
-  int ignored;
-  return ExtensionIndex(extension_nid, &ignored);
+  const StatusOr<int> index(ExtensionIndex(extension_nid));
+  if (index.ok()) {
+    return TRUE;
```

Comment:
Huh? This method has a `Cert::Status` return type, and is returning `Cert::TRUE`, I'm not sure I see where the `int` comes in?